internal/repositories/equipment_test.go

```diff
@@ -54,16 +54,17 @@ func (s *EquipmentSuite) SetupTest() {
 		t.Fatal(err)
 	}
 
+	name := "admin"
 	s.user = &ent.User{
-		Login: "admin", Email: "admin@email.com", Password: "12345", Name: "admin",
+		Login: "admin", Email: "admin@email.com", Password: "12345", Name: &name,
```

Comment:
Why creating a separate variable instead of assigning value directly in User struct?

Reply:
Because the Name filed has become a pointer. In the case of creating a pointer for the "admin" user value right inside the new User object it will be a helper function required which is more cumbersome in my opinion